mne/filter.py

```diff
@@ -508,7 +517,8 @@ def estimate_ringing_samples(system, max_try=100000):
 
 @verbose
 def construct_iir_filter(iir_params, f_pass=None, f_stop=None, sfreq=None,
-                         btype=None, return_copy=True, verbose=None):
+                         btype=None, return_copy=True, verbose=None, *,
+                         phase='zero'):
```

Comment:
By convention we always put `verbose` last
```suggestion
                         btype=None, return_copy=True, *, phase='zero',
                         verbose=None):
```